user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Low-Code_Apps_configuration/LowCodeApps_event_config.md

```diff
@@ -45,7 +45,9 @@ To configure an event to launch a script:
 1. Optionally, click the *Show settings* button to configure [script execution options](xref:Script_execution_options).
 
 > [!NOTE]
-> Data can often contain multiple values, and therefore the values will be wrapped in a JSON array. To be consistent, this will also be the case when only a single value is provided.
+>
+> - Data can often contain multiple values, and therefore the values will be wrapped in a JSON array. To be consistent, this will also be the case when only a single value is provided.
+> - From DataMiner 10.4.0 [CU11]/10.5.2 onwards<!--RN 41495-->, when an interactive Automation script is launched using the *Launch a script* event and the parameter *useNewIASInputComponents=true* has been added to the URL of the low-code app, the latest version of the interactive Automation script UI is displayed. If the parameter is set to "false" or omitted from the URL, the low-code app will use the old UI to display the interactive Automation script.
```

Comment:
I would not mention the option useNewIASInputComponents=true, because this page is about configuring an LCA. The option can be used when consuming the LCA. Whether the new IAS components are used or not, is not defined by a LCA configuration.